pkg/apis/controlplane/v1beta1/types.go

```diff
@@ -201,6 +201,9 @@ type NetworkPolicyRule struct {
 	// action “nil” defaults to Allow action, which would be the case for rules created for
 	// K8s Network Policy.
 	Action *secv1alpha1.RuleAction `json:"action,omitempty" protobuf:"bytes,6,opt,name=action,casttype=github.com/vmware-tanzu/antrea/pkg/apis/security/v1alpha1.RuleAction"`
+	// EnableLogging is used to indicate if agent should generate logs
+	// when rules are matched. Should be default to false.
```

Comment:
Feel better not to expose our internal implementation in API/CRD, so no need to mention agent here. Consider:
EnableLogging indicates whether or not to generate logs when rules are matched. Default to false.